docs/content/components/markdown.md

```diff
@@ -7,6 +7,284 @@ bundle: markdown
 ---
 
 
+```html live
+<div class="markdown-body">
+  <p>Text can be <b>bold</b>, <i>italic</i>, or <s>strikethrough</s>. <a href="https://github.com">Links</a> should be blue with no underlines (unless hovered over).</p>
```

Comment:
```suggestion
  <p>Text can be <b>bold</b>, <i>italic</i>, or <s>strikethrough</s>. <a href="https://github.com">Links </a>sould be blue with no underlines (unless hovered over).</p>
```

Reply:
looks like a missing space here 
<img width="274" alt="Screenshot 2020-04-13 17 05 12" src="https://user-images.githubusercontent.com/586552/79161235-1e542f00-7da9-11ea-86b6-6c10050a0e4e.png">

Reply:
Yeah, there is a problem with whitespace getting removed if text follows an HTML element, like `</div> some text` would render as `</div>some text`.

Also noticed it here:

![Screen Shot 2020-04-14 at 11 19 44 AM](https://user-images.githubusercontent.com/378023/79179340-65423300-7e42-11ea-8b27-895d32477aa2.png)

I'll make an issue in Doctocat.

Reply:
Ohh.. there is already an issue that I already forgot about: https://github.com/primer/doctocat/issues/105 😄